containers/ws/README.md

```diff
@@ -1,99 +1,153 @@
-# Cockpit on Fedora CoreOS or other container hosts
+# Cockpit WS Container
 
-The standard Fedora and Red Hat Enterprise Linux CoreOS images does not contain
-Cockpit packages.
+This container is suitable for scenarios where you can not or wish not to deploy the Cockpit Web Service and browser
+application natively.
 
-1. Install Cockpit packages as overlay RPMs:
+## Tags
+
+The following tags are provided for the image:
+
+- `:latest`, `:latest-amd64`, `:latest-arm64` point to the latest images and their respective architecture
+- `:{Cockpit WS Version}`, `:{Cockpit WS Version}-amd64`, `:{Cockpit WS Version}-arm64` (e. g. `:280-amd64`) point to
+  images with the specified Cockpit WS version and their respective platform
+
+## Preparing Managed Host
+
+The hosts you want to manage via this container need to have both `cockpit-system` installed and SSH access enabled.
+
+Depending on your configuration, you may want to install
+[additional extensions](https://packages.fedoraproject.org/search?query=cockpit-) as well, such as `cockpit-kdump` or
+`cockpit-networkmanager`.
```

Comment:
Please move this back down after the "install cockpit core packages" step, it IMHO makes more logical sense that way.